tests/test_mgxs_library_no_nuclides/results_true.dat

```diff
@@ -75,6 +75,15 @@
    material  group out nuclide  mean  std. dev.
 1     10000          1   total   1.0   0.046071
 0     10000          2   total   0.0   0.000000
+   material  group out nuclide      mean  std. dev.
+1     10000          1   total  0.991442   0.050935
```

Comment:
If I understand correctly, these results indicate that `ChiPrompt` is not properly normalized. Or does this mean that 0.991442 neutrons emitted from fission are prompt and in the fast group?